### DISCUSSION TITLE
Why this is implemented?

### DISCUSSION THREAD

**Discussion by SouvikUnscripted** [stegripe/rawon#491] [Quest and Answer] [answered]
![image](https://user-images.githubusercontent.com/40933469/135122700-9316cd58-617a-4bd4-884a-7c6e73953a11.png)
This should not be implemented I feel. - he can use the skip-to command without the DJ role. - Although, if there are multiple people in the voice channel. the vote will be used to skip to that particular song. Also, add DJ role direct skip in /skip command. The work of the DJ role will be only if someone comes with a DJ role and skip any song, the voting feature will not come, there will be a direct skip for both /skip command and /skip-to command. - Correct me if I am wrong.

**Comment by lxndr-rl** [comment]
Remember that version 2 is not finalized yet, it is still subject to revision and changes.
Although I must admit you have a good point.

**Comment by Mednoob** [comment, marked as answer]
@FireUpdate 

1. Skip-to command is designed to move to a specific position in the queue. That's why I don't add voting since there's a chance that: Even though all people voted, their vote can be different. For example: 1 people voted to skip to the last song, but there are 3 people voted to skip to the 3rd song. This may end up where all of the vote haven't reached the vote requirement (which is half of the VC members). That means voting in "skip-to" command isn't actually needed. They can just all vote together to skip manually. But, I will still consider about this.

2. DJ role direct skip in skip command is already implemented 4 days ago ([commit](https://github.com/zhycorp/disc-11/commit/9a080b75fb7352541a18b5154c6ae38f92a2f288#diff-e288500c2fe3208b28dcd240387c451d0971acf8f16290ded624e00a058394d7)). Direct skip can be done if they're the DJ, or if they're the music requester.
